android/quest/src/main/java/org/smartregister/fhircore/quest/ui/patient/profile/PatientProfileViewModel.kt

```diff
@@ -306,6 +306,13 @@ constructor(
               clientIdentifier = patientId,
               populationResources = profile.populationResources
             )
+          R.id.patient_change_status ->
```

Comment:
Remove the code for on-click handling for the old menu items.
i.e `R.id.remove_hiv_patient`.